### DISCUSSION TITLE
Storytelling with Power BI Bookmarks

### DISCUSSION THREAD

**Discussion by danmarshall** [microsoft/SandDance#589] [Show and tell]
Here's a quick video that shows how to use bookmarks in Power BI with the SandDance custom visual for data storytelling: (sound on 🔊)

https://user-images.githubusercontent.com/11507384/213285977-fa68273b-49aa-4fde-835c-e68af217b794.mp4

**Comment by danmarshall** [comment]
Note that we've deprecated the Snapshots button in favor of Bookmarks.

**Comment by Guiia** [comment]
Thank you for answering on the Power BI community.
It would be interesting to work for personal markers created by users on Power BI Service or reactivate the snapshots button so that users can also tell their stories.

**Reply by lova3211** [reply]
I also have some interesting ideas about this, but I take a few snapshots for this reason.
